spicy/toolchain/src/compiler/codegen/grammar-builder.cc

```diff
@@ -127,13 +127,20 @@ struct Visitor : public hilti::visitor::PreOrder<Production, Visitor> {
                 // Skipping not supported
             }
 
+            else if ( const auto& size = AttributeSet::find(n.attributes(), "&size") ||
```

Comment:
This means that e.g., hooks or other attached stateful computations (e.g., `&requires`, `&convert`) would not be invoked on skipped fields. We could document that with `skip` whether they are invoked is undefined.

Reply:
I updated the docs.